apps/studio/src/utils/sitemap.ts

```diff
@@ -23,14 +23,19 @@ const getSitemapTreeFromArray = (
   // Get the immediate children of the resource with the given parent ID
   const children = resources.filter((resource) => {
     if (parentId === null) {
-      return resource.parentId === null && resource.type !== "RootPage"
+      return (
+        resource.parentId === null &&
+        resource.type !== "RootPage" &&
+        resource.type !== "PageOrder"
+      )
     }
     return (
       resource.parentId === parentId &&
       resource.permalink !== INDEX_PAGE_PERMALINK
     )
   })
 
+  // TODO: Sort the children by the page ordering if the PageOrder resource exists
```

Comment:
does the order in the sitemap matter since we are already sorting this during publishing stage?

Reply:
This is used for the preview only, so we are maintaining the same behaviour as much as possible with what the user will see on the actual site.